src/main/java/hw6/entities/Data.java

```diff
@@ -0,0 +1,50 @@
+package hw6.entities;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Data {

```

Comment:
What kind of data is this ? You need to named class with referense to it purpose.

Reply:
Renamed